sphinx/transforms.py

```diff
@@ -556,6 +556,27 @@ def get_ref_key(node):
                 node['entries'] = new_entries
 
 
+class FigureTranslator(Transform):
+    """
+    Substitute language-specific figures for original one.  This transform looks
+    for translated images by filename.  The filename that is being looked for is
+    constructed as follow :samp:`filename.{language}.ext`.  For example, for a
```

Comment:
While the sample is nice in illustrating what happens here, it actually depends on the specific configuration value, no?